custom_components/tuya_local/devices/airrobo_p20_vacuum.yaml

```diff
@@ -64,6 +64,11 @@ entities:
             value: cleaning
           - dps_val: sleep
             value: idle
+      - id: 6

```

Comment:
This should not be added, as Home Assistant has deprecated battery_level for vacuum entities, and will remove them soon.